INSTALL.md

```diff
@@ -1,242 +1,190 @@
 # Installation and Configuration for cfgov-refresh
 
 There are two ways to install:
- - [Vagrant-box installation](INSTALL.md#vagrant-box-installation)
- - [Stand-alone installation](INSTALL.md#stand-alone-installation)
+- [Stand-alone installation](#stand-alone-installation)
+- [Vagrant-box installation](#vagrant-box-installation)
+  :warning: These instructions are not currently working.
```

Comment:
It's failing for me because it installs MySQL 5.1 and (default_storage_engine)[https://github.com/cfpb/cfgov-refresh/blob/refactor-setup/.env_SAMPLE#L45] is a setting that was introduced in 5.5.